src/App.js

```diff
@@ -6,7 +6,7 @@ import { BrowserRouter as Router } from "react-router-dom";
 import "./App.css";
 import Footer from "./components/Footer/Footer";
 import Navbar from "./components/Navbar/Navbar";
-import HomePage from "./containers/HomePage/HomePage";
+import HomePage from "./containers/HomePage/HomePage"; // eslint-disable-next-line
```

Comment:
we dont need to disable eslint for next line since it needs to be used on line 19